frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  Avatar as ChakraAvatar,
+  AvatarBadge as ChakraAvatarBadge,
+  AvatarProps as ChakraAvatarProps,
+} from '@chakra-ui/react'
+
+export interface AvatarProps extends ChakraAvatarProps {
+  hasNotification?: boolean
+}
+
+/*
+ * Avatar component
+ * May be wrapped in AvatarMenuButton
+ */
+export const Avatar = ({
+  hasNotification,
+  ...rest
+}: AvatarProps): JSX.Element => {
+  const avatarBadge = hasNotification ? <ChakraAvatarBadge /> : ''
+
+  return (
+    <ChakraAvatar
+      getInitials={
+        rest.getInitials ? rest.getInitials : (fullName) => fullName[0] // Extract first letter of avatar name
```

Comment:
I think this is done by default? I don't think we need to override this behaviour

Reply:
Need, because default is to extract two characters

Reply:
Added docs for clarity